modules/aws-outposts-requirements-limitations.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="aws-outposts-requirements-limitations_{context}"]
+= AWS Outposts on {product-title} requirements and limitations
+
+By configuring your {product-title} cluster to accommodate the requirements and limitations of this platform, you can manage the resources on the AWS Outpost similarly to a cloud-based AWS cluster.
+
+* To extend an {product-title} cluster on AWS into an AWS Outpost, you must have installed the cluster into an existing VPC.
+
+* {product-title} clusters on AWS include the `gp3-csi` and `gp2-csi` storage classes.
+These classes correspond to Amazon Elastic Block Store (EBS) gp3 and gp2 volumes.
+{product-title} clusters use the `gp3-csi` storage class by default, but AWS Outposts does not support EBS gp3 volumes.
+You must configure the compute machine set that creates machines in your AWS Outposts instance to use gp2 volumes.
++
+For more information about AWS general-purpose EBS volume types, see the AWS documentation link:https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-plan-storage-compare-volume-types.html[Comparing Amazon EBS volume types gp2 and gp3].
+
+////
+//The existing limitation text has these `oc` commands to change the default storage class. Do we need this, or can we just have the machineset use gp2 for machines it creates?
```

Comment:
I think the procedure is also about persistent volume claims, so we need to cover that somewhere, but, machineset setting gp2 is enough for getting machines up